### PR TITLE
Spring pagination

### PR DESCRIPTION
**Pull request by fedaktaras**

### CODE REVIEW

**Review by EduarfF — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -6,7 +6,7 @@
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-starter-parent</artifactId>
         <version>3.2.4</version>
-        <relativePath/>
+        <relativePath/> <!-- lookup parent from repository -->
```
Comment (andrii-hoienko) [inline pom.xml]:
```suggestion
        <relativePath/>
```

File: pom.xml [hunk 2]
```diff
@@ -84,6 +83,30 @@
...
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-validation</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.springdoc</groupId>
+            <artifactId>springdoc-openapi-starter-webmvc-ui</artifactId>
+            <version>2.1.0</version>
+        </dependency>
```
Comment (andrii-hoienko) [inline pom.xml]: Please be consistent and use properties to define versions of dependencies

**Review by andrii-hoienko — CHANGES_REQUESTED** [review]
Nice, left minor comments

**Review by andrii-hoienko — APPROVED** [review]
Good job!

**Pull request merged**